influxql/ast.go

```diff
@@ -598,7 +598,7 @@ type SelectStatement struct {
 	groupByInterval time.Duration
 
 	// if it's a query for raw data values (i.e. not an aggregate)
-	RawQuery bool
+	IsRawQuery bool
```

Comment:
Idk, is it? I realized that it was a bool and since it's not in a struct that contains nothing but bools, I figured it would be clearer to have the `Is` there. Curious to hear what others think. @benbjohnson, @dgnorton?